ibis-server/app/mdl/rewriter.py

```diff
@@ -44,7 +44,7 @@ def rewrite(self, sql: str) -> str:
 
     def _transpile(self, planned_sql: str) -> str:
         write = self._get_write_dialect(self.data_source)
-        return sqlglot.transpile(planned_sql, read="trino", write=write)[0]
+        return sqlglot.transpile(planned_sql, write=write)[0]
```

Comment:
This method is used for v2 and v3. For v2, the read dialect is `trino`. How about splitting this method for each class?

Reply:
hmm..  No, maybe you're right. We're better to split v2 and v3 because the trino dialect also handles the functions converting between trino and others data source ( Currently, the v2 function white list used by AI service is based on the trion function) 🤔